modules/helper/VCenterHelper.py

```diff
@@ -0,0 +1,31 @@
+from interfaces.host import Host
+from modules.helper.GeneralHelper import GeneralHelper
+from modules.api.VcenterConnection import VcenterConnection
+
+from typing import List
+
+
+class VCenterHelper:
+
+    def __init__(self, vcenter_username: str, vcenter_password) -> None:
+        self.general_helper = GeneralHelper()
+        self.vcenter_username = vcenter_username
+        self.vcenter_password = vcenter_password
+
+    def get_esxi_overall_stats(self) -> List[Host]:
```

Comment:
This function should be renamed (even though it is too long, but couldn't come up with something better):
` get_esxi_overall_stats_for_all_vcenters`
I would expect, this is for one vcenter only. Only the `vc_conn.get_esxi_overall_stats()` is doing the difference an named the same?